lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -150,6 +158,7 @@ public function getDateTimeFormat($type)
     }
 
     /**
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity)
```

Comment:
Refactor bad smelling code instead of suppressing warning whenever possible.

Reply:
This annotation needs to be removed and corresponding code refactored (unlike the `PHPMD.CouplingBetweenObjects` it should be pretty easy).